app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="menu_item_title_flag_as_favorite">お気に入りに追加</string>
+    <string name="menu_item_title_unflag_as_favorite">お気に入りから削除</string>
+    <string name="menu_item_title_delete_favorite">削除</string>
+    <string name="menu_item_title_set_alarm">アラーム設定</string>
+    <string name="menu_item_title_delete_alarm">アラーム削除</string>
+    <string name="menu_item_title_delete_all">全て削除</string>
+    <string name="menu_item_title_favorites">お気に入り</string>
+    <string name="menu_item_title_feedback">フィードバック</string>
+    <string name="menu_item_title_navigate">ナビゲート</string>
+    <string name="menu_item_title_refresh">リフレッシュ</string>
+    <string name="menu_item_title_share_session">イベントを共有</string>
+    <string name="menu_item_title_share_favorites">お気に入りを共有</string>
+    <string name="OK">OK</string>
+    <string name="certificate_error_message">"         安全な接続の確立に失敗しました。          (<xliff:g example="Unknown Error" id="error">%s</xliff:g>)     "</string>
+    <string name="certificate_error_title">証明書を追加できませんでした。</string>
+    <string name="dlg_err_connection_failed">接続失敗</string>
+    <string name="dlg_err_failed_unknown_host">不明なアドレス：％s</string>
+    <string name="dlg_err_failed_wrong_http_credentials">無効なHTTP認証</string>
+    <string name="dlg_err_failed_timeout">サーバーが応答しません（タイムアウト）。</string>
+    <string name="dlg_err_failed_connect_failure">不明な接続失敗</string>
+    <string name="dlg_err_failed_not_found">無効なスケジュールURL</string>
+    <string name="dlg_err_failed_parse_failure">応答を解析できませんでした</string>
+    <string name="dlg_err_failed_ssl_failure">SSL障害</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HTTPは許可されていません。 HTTPSを使用してください。</string>
+    <string name="uptodate">スケジュールは最新です。</string>
+    <string name="schedule_changes_dialog_title">スケジュールを更新</string>
+    <string name="schedule_changes_dialog_updated_to_text">スケジュールが更新されました。</string>
+    <string name="schedule_changes_dialog_changed_new_cancelled_text">. ここに変更があります<xliff:g example="3 lectures" id="num_changes">%1$s</xliff:g>, <xliff:g example="2 are" id="num_new">%2$s</xliff:g> 新規, <xliff:g example="1 is" id="num_canceled">%3$s</xliff:g> キャンセルした.</string>
+    <string name="schedule_changes_dialog_affected_text">変更は、お気に入りの講義の<xliff:g example="2" id="num_marked">%1$d</xliff:g>に影響します。</string>
+    <string name="schedule_changes_dialog_browse">ブラウズ</string>
+    <string name="schedule_changes_dialog_later">後で</string>
+    <string name="progress_loading_data">&#8230;スケジュールを取得</string>
+    <string name="progress_processing_data">&#8230;スケジュールを処理中</string>
+    <string name="load_data">ロード</string>
+    <string name="choose_day">日付を選択</string>
+    <string name="day">日付</string>
+    <string name="fahrplan">スケジュール</string>
+    <string name="appVersion">アプリバージョン</string>
+    <string name="schedule_updated_to">"         <xliff:g example="Mildenberg 2018-01-16 16:50" id="version">%s</xliff:g>に更新しました。"</string>
```

Comment:
Please remove the surrounding quotes and whitespace. It might have been added by the editor.

Reply:
Thanks for removing the quotes. The whitespace can be removed, too.